patches/minecraft/net/minecraft/client/renderer/BlockModelShapes.java.patch

```diff
@@ -1,5 +1,20 @@
 --- ../src-base/minecraft/net/minecraft/client/renderer/BlockModelShapes.java
 +++ ../src-work/minecraft/net/minecraft/client/renderer/BlockModelShapes.java
+@@ -201,12 +201,12 @@
+             ibakedmodel = this.field_178128_c.func_174951_a();
+         }
+ 
+-        return ibakedmodel.func_177554_e();
++        return ibakedmodel.getParticleTexture(p_178122_1_);
+     }
+ 
+     public IBakedModel func_178125_b(IBlockState p_178125_1_)
+     {
+-        IBakedModel ibakedmodel = this.field_178129_a.get(p_178125_1_);
++        IBakedModel ibakedmodel = this.field_178129_a.get(p_178125_1_.getClean());
```

Comment:
Probably better to follow the pattern of the rest of the places and explicitly check if its an extended state. Adding functions after the fact is kinda wonkey in java.